### PR TITLE
Move default frontend theme

### PR DESCRIPTION
**Pull request by akx**

### CODE REVIEW

**Comment by Pikkupomo** [comment]
This also seems quite a big change and without further discussions I'm not that interested in to merging this.

I think this is a good thing to do, and could simplify our template structures, but I would like to see input from @suutari-ai and @juhakujala

**Comment by akx** [comment]
Sure, we can keep this on hold until Further Discussions occur :)

**Comment by Pikkupomo** [comment]
We can merge this, any objections @suutari-ai?

**Comment by akx** [comment]
Rebased on d56dd24954e2921e3d4934a0e56387c84fbf4192

File: branding/shoop_logo_only.svg [hunk 1]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" height="1985" width="1985"><defs><clipPath id="a"><circle r="992" cy="992" cx="992"/></clipPath></defs><circle r="992" cy="992" cx="992" fill="#1bab95"/><path clip-path="url(#a)" d="M1436.502 355.512L533.336 498.785v542.68L861.62 1369.72 465.5 1593.51l397.107 393.007 1116.107-11.013c0-30.237 10.208-1054.745 7.557-1084.294z" fill="#17907d"/><path d="M936.31 511.387c.09 32.09.183 64.18.274 96.268h500.766V356.398c-225.075.807-450.214-1.666-675.246 1.333-106.022 3.94-225.723 43.89-273.074 147.278-50.426 106.126-42.21 227.55-36.2 341.678 8.19 98.885 55.278 204.434 151.92 244.722 116.764 51.967 246.81 32.976 370.813 36.71h144.092c-.49-35.122-.982-70.244-1.473-105.365l269.44 222.04-269.44 235.497v-100.93c-217.853-6.475-435.65-15.13-653.282-26.91v242.283c255.997 19.84 512.692 36.406 769.555 35.167 104.486-2.236 222.213-41.453 269.732-142.53 50.638-102.356 38.175-219.444 37.728-329.98-3.492-103.973-47.213-219.18-149.015-262.92-111.582-50.82-236.435-31.735-355.204-35.56H936.01l.3 103.282-262.707-222.04C761.17 663.9 848.745 587.625 936.31 511.387z" fill="#fff"/></svg>
```
Comment (suutari-ai) [inline branding/shoop_logo_only.svg]: Why this is changed?

Comment (suutari-ai) [inline branding/shoop_logo_only.svg]: Oops, sorry.. I missed the commit message.

File: shoop/front/template_helpers/urls.py [hunk 2]
```diff
@@ -31,8 +31,21 @@ def model_url(context, model, absolute=False):
...
+    :return: Reversed URL or None
+    :rtype: str|None
+    """
     try:
-        reverse(url, args=args, kwargs=kwargs)
-        return True
+        return reverse(url, args=args, kwargs=kwargs)
     except NoReverseMatch:
-        return False
+        return None
```
Comment (suutari-ai) [inline shoop/front/template_helpers/urls.py]: has_url should probably be named get_url or something like that when it is used for getting the url.

**Pull request merged**